edi_voxel_account_invoice_oca/tests/test_voxel_account_invoice.py

```diff
@@ -218,7 +222,19 @@ def _get_client_data(self):
         }
 
     def _get_customers_data(self):
-        return []
+        return [
+            {
+                "SupplierClientID": "C01",
```

Comment:
Why is this changing as well?

Reply:
Another default value, field "partner_shipping_id": False,